sys/vm/vm_map.c

```diff
@@ -3326,7 +3326,7 @@ vm_map_protect(vm_map_t map, vm_offset_t start, vm_offset_t end,
 
 		/*
 		 * When VM_MAP_PROTECT_KEEP_CAP is used (for mprotect()),
```

Comment:
Suggestions for commit log:  s/mprotect generally didn't do/mprotect generally doesn't do/ and s/current permission were/current permissions were/